File: C/Ex18.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){

  float peso, altura, imc;
  char sexo;

  printf("Digite o sexo(M ou F): ");
  scanf("%c", &sexo);
  
  printf("Digite o peso: ");
  scanf("%f", &peso);

  printf("Digite a altura: ");
  scanf("%f", &altura);

  imc = peso/(altura*altura);

  if(sexo == 'F'){
    if(imc<19){
      printf("Abaixo do peso");
    }  
    else if(imc>=19 && imc <24) {
      printf("Peso ideal");
    }
    else{
      printf("Acima do peso");
    }  
    printf("\n seu imc e %.2f", imc);
  }

  else if(sexo == 'M'){
    if(imc<20){
      printf("Abaixo do peso");
    }  
    else if(imc>=20 && imc <25) {
      printf("Peso ideal");
    }
    else{
      printf("Acima do peso");
    }  
    printf("\n seu imc e %.2f", imc);
  }
else
    printf("Digite conforme solicitado.");

  return 0;
}

File: Python/Ex18.py
peso = float(input("Digite o peso: "))
altura = float(input("Digite a altura: "))
sexo = str(input("Digite o sexo: F femenino, M masculino \n"))
imc = peso/(altura*altura)
 
if (sexo == 'F' or sexo =='f'):
        if(imc <19):
            print("Abaixo do peso") 

elif(sexo == 'F' or sexo == 'f' and imc >= 19 and imc <24):
    print("Peso ideal")
elif(sexo == 'F' or sexo =='f' and imc >= 19 or imc >24):
    print("Acima do peso")



elif (sexo == 'M' or sexo =='m'):
        if(imc <20):
            print("Abaixo do peso") 

elif(sexo == 'M' or sexo =='m' and imc >= 20 and imc <25):
    print("Peso ideal")
else:
    print("Acima do peso")